modules/claims_api/config/schemas/v2/request_bodies/disability_compensation/example.json

```diff
@@ -1,5 +1,6 @@
 {
   "data": {
+    "id": "42ba67b9-36ce-4e30-9d7a-fd1155452f89",
```

Comment:
I think this is the line that needs to come out since this example file represents an example being sent in, in which case they would not have an ID.